torch/_decomp/decompositions.py

```diff
@@ -835,6 +835,7 @@ def native_dropout(input: Tensor, p: float, train: Optional[bool]):
 
 @register_decomposition(aten._softmax)
 def _softmax(x: Tensor, dim: int, half_to_float: bool):
+    x = x.contiguous()
```

Comment:
can you leave a comment here that this is to match eager behavior?

Reply:
Done